mirrulations-client/src/mirrclient/client.py

```diff
@@ -149,15 +149,21 @@ def get_job(self):
         :raises: NoJobsAvailableException
             If no job is available from the work server
         """
-        print('Staring New Job')
+        print('Starting New Job')
         response = requests.get(f'{self.url}/get_job',
                                 params={'client_id': self.client_id},
                                 timeout=10)
 
         job = loads(response.text)
+        link = 'https://www.regulations.gov/'
         if 'error' in job:
             raise NoJobsAvailableException()
-        print(f'Job ID: {job["job_id"]}')
+        if 'docket' in str(job['url']):
+            print(f'Regulations.gov link: {link}docket/{job["url"][39:]}')
+        if 'document' in str(job['url']):
+            print(f'Regulations.gov link: {link}document/{job["url"][39:]}')
+        if 'comment' in str(job['url']):
+            print(f'Regulations.gov link: {link}comment/{job["url"][39:]}')
```

Comment:
39?  What is this magic number for?  At a minimum, include a comment - but I have to believe there is a better way to get this url.

Reply:
Just took a quick look at this and here is the difference:
**With Magic number output:**
Regulations.gov link: https://www.regulations.gov/docket/HHS-2007-0200
**Without magic number output:**
Regulations.gov link: https://www.regulations.gov/docket/https://api.regulations.gov/v4/dockets/HHS-2007-0200

- Magic number removes the need for an API key to directly access the link
- Essentially normalizes the link

@jack11wagner can you confirm this?

Reply:
Would `print(f'Regulations.gov link: {job["url"]}')` work?  

`link = 'https://www.regulations.gov/'` and the `job["url"]` also contains that prefix.  

I think we can eliminate `link` entirely.

At the point, the entire `if` structure might go away!

Reply:
`job["url"]` gives a link that requires the API key: `https://api.regulations.gov/v4/dockets/HHS-2007-0200`. When trying to access the link, it cannot be seen without an API key. 

The purpose of this section of code is to normalize the link. It does this by giving the normal Regulations.gov link (link = 'https://www.regulations.gov/'), appending the type (docket, document, comment), and ends with appending the type ID (HHS-2007-0200). This gives the following normalized link: `https://www.regulations.gov/docket/HHS-2007-0200`. 

I will add comments regarding how this works. 

Do you believe it would be better to print the API link or normalize it?